vng_api_common/utils.py

```diff
@@ -203,3 +203,9 @@ def get_view_summary(view_cls):
         pass
 
     return ""
+
+
+def get_field_attribute(model_string: str, field_name: str, attr_name: str) -> str:
```

Comment:
function can return `None`

```suggestion
def get_field_attribute(model_string: str, field_name: str, attr_name: str) -> Optional[str]:
```